src/gnome-mastermind.c

```diff
@@ -1546,13 +1566,8 @@ static gboolean tray_mid_click(){
 
 //	 gm_debug ("i:%d ypos:%d c: %d l:%d\n", i, ypos, c, l);
 
-      if (timeout_id > 0) {
```

Comment:
The purpose of this block was to not reset the tray area if you click the middle button fast multiple times. This way the peg gets big on the first click and only goes back to normal after the last one. With this change the peg can be reset at random times with respect to the last click. No idea why this was removed but I guess I'll restore the old behaviour as it has nothing to do with the GTK3 port.